src/schematools/contrib/django/models.py

```diff
@@ -162,6 +163,32 @@ def table_schema(cls) -> DatasetTableSchema:
         """Give access to the original table_schema that this model implements."""
         return cls._table_schema
 
+    @classmethod
+    def get_field_schema(cls, model_field: Union[models.Field, models.ForeignObjectRel]):
+        """Provide access to the underlying amsterdam schema field that created the model field."""
+        if isinstance(model_field, models.ForeignObjectRel):
+            # When Django auto-creates the related field, it doesn't have `field_schema`,
+            # but it can be resolved by looking up the original forward relation.
+            return model_field.remote_field.field_schema
```

Comment:
is it guaranteed that the forward relation has the `field_schema` property? or do we need the same exception handler as in the else-branch ?

Reply:
Good point, I had my doubts too. Updated the PR.